api/src/main/java/org/openmrs/api/db/hibernate/HibernateProgramWorkflowDAO.java

```diff
@@ -465,7 +465,7 @@ public List<PatientProgram> getPatientProgramByAttributeNameAndValue(String attr
 		Query query;
 		try {
 			query = sessionFactory.getCurrentSession().createQuery(
-					"SELECT pp FROM patient_program pp " +
+					"SELECT pp FROM PatientProgram pp " +
```

Comment:
Was this change also related to the ticket you were working on?

Reply:
yes, because after my changes, some other tests were failing.

Reply:
Would this mean that it was some sort of bug that your tests exposed?

Reply:
No! for this particular case, my task was to implement tests for this method `getPatientProgramByAttributeNameAndValue` from `ProgramWorkflowService` class. But after implementing the test,  I faced an error related to the table mappings `patient_program pp is not mapped [select pp from` ....
So I applied this changes to deal with the related issue.

Reply:
I still do not get it. If it was not a bug, why do you need to fix it?

Reply:
It was, but not related to any code added by me. The `sessionfactory` query was there, but was not covered by the tests. once I wrote a test, the error came up.

Reply:
Now, that makes perfect sense. Thanks for the clarification. 😊

Reply:
Thank for your time. 
I will be pushing the last changes (naming conventions) shortly